components/facebook_groups/README.md

```diff
@@ -0,0 +1,7 @@
+# Getting Started
+You need to install the Pipedream app in your Facebook Group in order to make any API calls, or use any prebuilt triggers or actions. To install Pipedream, navigate to the group for which you are an administrator, and in the left sidebar, under **Settings** > **Group Settings**, you will find **Manage Advanced Settings**. Click the Edit button under **Apps**.
```

Comment:
```suggestion
You need to install the Pipedream app in your Facebook Group to make any API calls, or use any prebuilt triggers or actions. To install Pipedream, navigate to the group where you're an administrator, and in the left sidebar, under **Settings** > **Group Settings**, you'll find **Manage Advanced Settings**. Click the Edit button under **Apps**.
```

Reply:
Changed 'in order to' to 'to', 'you are' to 'you're', 'will find' to 'you'll find' - Google's Developer docs style guide (contractions, future tense, wordiness)